### ISSUE TITLE
[FlexibleColumnLayout]: Missing screen reader instruction when focusing the separator in FlexibleColumnLayout

### ISSUE THREAD

**Issue by AlexYanSap** [SAP/ui5-webcomponents#11392]
### Describe the bug

When using FlexibleColumnLayout, if a user navigates to the separator between columns using keyboard navigation (e.g. Tab), the screen reader does not announce any instruction for how to interact with the separator.

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/layouts-floorplans-flexiblecolumnlayout--docs

### Reproduction steps

Use Tab to focus on the separator between columns.

Observe the screen reader output.

No instruction or hint is announced.

### Expected Behaviour

Screen reader should announce the interaction instructions when the separator is focused.(the user should use the arrow keys to adjust the separator)

### UI5 Web Components for React Version

2.9.0

### UI5 Web Components Version

2.9.0

### Browser

Chrome

### Relevant log output

```shell

```

### Organization

SAP SF

### Declaration

- [x] I’m not disclosing any internal or sensitive information.
